content/en/smart-contracts/wallets/transfer-fil/index.md

```diff
@@ -13,72 +13,44 @@ weight: 100
 
-### Sending to an Eth-style (0x) address
+## Filecoin-style account
 
-#### Using a Ledger hardware wallet
+The exact process to send FIL from a Filecoin-style (`f1`, `f2`, `f3`)  account on the Filecoin network depends on the destination. If your destination is:
 
-NOTE: Currently Ledger Live does not support `0x` or `f4` addresses, so you cannot use it to directly send funds to a `0x` address.
+- **Another Filecoin-style (`f1`, `f2`, `f3`) address**, use a Filecoin wallet, such as [Glif](https://glif.io) or [Ledger Live](https://www.ledger.com/ledger-live). Just enter the f-style address and make the transfer as per the wallet instructions.
 
-Glif wallet supports both address styles, and can send from an `f1` address to a `0x` address. So you can connect your Ledger device to a Glif wallet. You get the benefit of Glif's functionality, with your keys remaining on your Ledger hardware wallet.
+- **An Ethereum-style (`0x`) address**, you can:
+  - Use one of the methods described in [Transfer Fil to a 0x address]({{< relref "fil-to-zero-x" >}}) based on your specific situation.
+  - Use one of the [supported exchanges](#exchanges-that-support-0x).
 
-1. Ensure your Ledger device is connected to your computer and the Filecoin app is open on your hardware wallet and it says "Filecoin ready"
-2. Go to [Glif](https://glif.io) and click "Connect Wallet" at the top of the screen
-3. Select "Ledger (Filecoin)"
-4. Click "Send FIL" at the bottom of the screen
-5. Enter the `0x` address you with to send to and the amount
-6. Click send
-7. Verify and accept the transaction on your hardware device
+## Exchanges that support 0x
 
-#### Using a Glif 'burner' wallet
+If you are funding a Filecoin Ethereum-style (`0x`) account from an exchange, you must use one that supports this account type account. The following exchanges support withdrawing directly to Filecoin Ethereum-style addresses:
 
-Burner wallets are intended for temporary use, and keys are generated in your browser. This is less secure than a hardware wallet, but very useful for small transfers of FIL
+  - [Anchorage](https://www.anchorage.com/)
+  - [Bittrex](https://bittrex.com/)
+  - [Btcturk](https://www.btcturk.com/)
+  - [Coinbase](https://www.coinbase.com/)
+  - [Gemini](https://www.gemini.com/)
+  - [Huobi](https://www.huobi.com/)
+  - [Kraken](https://www.kraken.com/)
 
-1. Go to [Glif](https://glif.io) and click "Connect Wallet" at the top of the screen
-2. Select "Burner Wallet"
-3. You can import a seed phrase, or private key, or select "Create seed phrase" to create a new wallet
-4. Follow the instructions to create the wallet, noting down the seed phrase is you are creating a new one
-5. Click "Send FIL" at the bottom of the screen
-6. Enter the `0x` address you with to send to and the amount
-7. Click send
-8. Verify and accept the transaction on your hardware device
- 
+  </br>
```

Comment:
Add a sentence here saying something like:

> Any exchange that trades Filecoin but _is not_ in the above list is only able to send FIL directly to an`f3` address.

Reply:
Got it, added in [062e4bf](https://github.com/filecoin-project/filecoin-docs/pull/1820/commits/062e4bf3152072143b534614439262e8d5173c77)